최주현/3주차_01926.java

```diff
@@ -0,0 +1,88 @@
+import java.awt.*;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.LinkedList;
+import java.util.Queue;
+import java.util.StringTokenizer;
+
+//참고 블로그: https://blog.encrypted.gg/941
+
+public class Main {
+
+    static int[][] board;
+    static boolean[][] vis;
+    static int[] dx = new int[]{1, 0, -1, 0};
+    static int[] dy = new int[]{0, 1, 0, -1};
+    static int n, m, largest;
+
+    static void bfs(Point start){
+        //Point 클래스는 x, y 좌표로 구성되어 있음
+        //xy좌표를 활용하는 bfs에서 주로 Queue를 Point 타입으로 선언해서 활용함
+
+        //시작점을 Queue에 추가하고 시작
+        Queue<Point> queue = new LinkedList<>();
+        queue.add(start);
+        vis[start.x][start.y] = true;
```

Comment:
```suggestion
        visited[start.x][start.y] = true;
```
vis 보다는 visited, 혹은 visit이 조금 더 직관적일 것 같아요